From 1539089bcd70852304a630e56516f21645d94484 Mon Sep 17 00:00:00 2001
From: SidneyAn <ran1.an@intel.com>
Date: Fri, 3 Apr 2020 06:51:41 +0000
Subject: using el7 python3 dep pkgs

using el7 pkgs as python3 dep instead of el8 ones
unit tests are also disabled to reduce the python3 deps
which is not supported by el7

Signed-off-by: SidneyAn <ran1.an@intel.com>
---
 SPECS/python-daemon.spec | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/SPECS/python-daemon.spec b/SPECS/python-daemon.spec
index 97e1cb2..686228e 100644
--- a/SPECS/python-daemon.spec
+++ b/SPECS/python-daemon.spec
@@ -1,7 +1,7 @@
 %{?python_enable_dependency_generator}
 Name:           python-daemon
 Version:        2.2.3
-Release:        7%{?dist}
+Release:        7.el7%{?_tis_dist}.%{tis_patch_ver}
 Summary:        Library to implement a well-behaved Unix daemon process
 
 # Some build scripts and test franework are licensed GPLv3+ but htose aren't shipped
@@ -15,11 +15,11 @@ Patch1:         0001-Mock-the-.__class__-attribute-so-tests-pass-in-py2.patch
 
 BuildArch:      noarch
 BuildRequires:  python3-devel, python3-setuptools
-BuildRequires:  python3-testscenarios
-BuildRequires:  python3-docutils
+#BuildRequires:  python3-testscenarios
+BuildRequires:  python36-docutils
 BuildRequires:  python3-lockfile
-BuildRequires:  python3-mock
-BuildRequires:  python3-testtools
+BuildRequires:  python36-mock
+#BuildRequires:  python3-testtools
 
 %global _description\
 This library implements the well-behaved daemon specification of PEP 3143,\
@@ -32,8 +32,8 @@ This is the python2 version of the library.
 %package -n python3-daemon
 Summary:        Library to implement a well-behaved Unix daemon process
 Requires:       python3-lockfile
-Requires:       python3-docutils
-%{?python_provide:%python_provide python3-daemon}
+Requires:       python36-docutils
+%{?python_provide:%python_provide python%{python3_pkgversion}-daemon}
 
 %description -n python3-daemon %_description
 
@@ -51,7 +51,7 @@ rm -fr %{buildroot}%{python3_sitelib}/tests
 
 # Test suite requires minimock and lockfile
 %check
-PYTHONPATH=$(pwd) %{__python3} -m unittest discover
+#PYTHONPATH=$(pwd) %{__python3} -m unittest discover
 
 %files -n python3-daemon
 %license LICENSE.ASF-2
-- 
2.7.4

